IncrementSlider/IncrementSlider.js

```diff
@@ -103,6 +103,15 @@ const IncrementSliderBase = kind({
 		 */
 		backgroundProgress: PropTypes.number,
 
+		/**
+		 * The size of the buttons.
+		 *
+		 * @type {('small'|'large')}
+		 * @default 'large'
+		 * @public
+		 */
+		buttonSize: PropTypes.oneOf(['small', 'large']),
```

Comment:
We have the `decrementIcon` and the `incrementIcon` for the + and - buttons.
How about renaming `buttonSize` as `iconSize`?